index.html

```diff
@@ -92,7 +92,7 @@ <h2>Jump into using Caliper with the other projects</h2>
         <strong>Ethereum</strong>
         <img src="https://avatars1.githubusercontent.com/u/6250754?s=200&v=4" alt="Ethereum">
       </a>
-      <a class="project" href="{{ site.baseurl }}/{{ site.caliper.current_release }}/fabric-config">
+      <a class="project" href="{{ site.baseurl }}/{{ site.caliper.current_release }}/fabric-config/new/">
```

Comment:
This is not in the scope of this PR

Reply:
Yes, I couldn't figure out how to remove that, similar to something that happened with #1194

Reply:
Since that modification is already merged, it probably won't cause any issues. But check out the new Contributing docs in the repo to see how to avoid such issues.